Python/initconfig.c

```diff
@@ -1615,6 +1622,44 @@ config_read_env_vars(PyConfig *config)
     return _PyStatus_OK();
 }
 
+static PyStatus
+config_init_cpu_count(PyConfig *config)
+{
+    const char *env = config_get_env(config, "PYTHONCPUCOUNT");
+    if (env) {
+        int cpu_count = -1;
+        if (_Py_str_to_int(env, &cpu_count) != 0) {
+            cpu_count = -1;
+        }
+        if (cpu_count >= 1) {
+            config->cpu_count = cpu_count;
```

Comment:
What about raising an error, rather than silently ignoring invalid value?

For consistency, you should accept default.